### ISSUE TITLE
Parse Error when processing specific LaTeX equations

### ISSUE THREAD

**Issue by noze12** [djade007/latext#18]
Hello, thank you for the convenient package.

I have encountered a case where it shows a "Parse Error".  When passing text like the following:
```
$$\left\{\begin{array}{l}3 x-4 y=1 \\ -3 x+7 y=5\end{array}\right.$$
```
it results in a Parse Error.
<img width="362" alt="スクリーンショット 2024-04-07 13 39 28" src="https://github.com/djade007/latext/assets/13155037/6ddf5569-6015-487d-9d2f-9b6a33a8f36b">

The flutter_math online demo displays this TeX correctly.
<img width="897" alt="スクリーンショット 2024-04-07 13 48 40" src="https://github.com/djade007/latext/assets/13155037/e64e0335-51a8-43e7-a67c-a8139eff3537">

I suspect the issue might be occurring due to the split here.
https://github.com/djade007/latext/blob/a90876619809b8e1098c59262cfb39a0f6adb40d/lib/latext.dart#L175

I would be grateful if you could address this issue.

**Comment by djade007** [comment]
Fixed in #19 version `0.4.0`

**Comment by noze12** [comment]
Thank you! I've confirmed that the issue has been resolved with the update.
